rosidl_generator_cpp/resource/msg__struct.hpp.template

```diff
@@ -107,8 +107,7 @@ for field in spec.fields:
 
   // field types and members
 @[for field in spec.fields]@
-  typedef @(msg_type_to_cpp(field.type))
-    _@(field.name)_type;
+  using _@(field.name)_type = @(msg_type_to_cpp(field.type));
```

Comment:
Please keep the newline. For the new line it should be after the equal sign.

Same below.